.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+<!-- - Spend some time on your titles 
+  - Make them alluring. 
+  - Short without too much detail 
+  - The issues are viewed in some sort of list view 80% of the time. -->
+As issues are viewed in some sort of list view 80% of the time (or so) the title is the part being read most often. 
+It can therefore be a good idea to spend some time formulating one. 
+
+A good title is short without too much detail. Details should be presented in the actual description. 
+
+If possible try to make the title compelling. While an open-source repository is not a newspaper where readers must be enganged at any cost, you still want to catch the interest of contributors. 
+
+The following title: 
+> Input causes error when you type over 50 characters and press enter 
+
+could be improved by writing: 
+> Input error on 50-character submit
+
+#### The good issue is easily readable
+<!-- - Readability 
+- Easily understandable 
+  - No internal lingo 
+- Proof-read --> 
+
+There are MANY ways to improve the readability of a text. The following is just some ways to do it, that we think can add value when writing an issue:
+- Using proper punctuation (to the best of your abilities at least. We are not experts at ,it either)
+- Spending some time proof-reading the issue before it is submitted. 
+- Using available formatting such as lists and code blocks to aid easy skimming. 
+- Explaining the abbrevations being used 
+
+<!-- Perhaps come up with a more code related example - i just couldn't at the time of writing -->
+For example the text: 
+> I went to the RDL the other day, i think that the RDL is very nice! They have a huge selectin of books such as: the old man and the sea, 1984, Brave new world and the count of monte cristo 
+
+Is much harder to read than: 
+> I went to the royal danish library (RDL) the other day. The RDL is very nice. They have a huge selection of books, such as: 
+> - the old man and the sea 
+> - 1984 
+> - brave new world 
+> - the count of monte cristo
+
+#### The good issue contains screenshots and gifs if possible 
+<!-- - A picture says more than 1000 words... (Imagine how much a gif is saying) --> 
+Kirby is a design system, hence it is a very visual project! So do not be shy of using images and gifs to explain yourself. 
+
+If you are requesting a new component, perhaps add a sketch showing how you are imagining it. When reporting a bug, record a gif to show exactly what you are experiencing. 
+Suggesting an enhancement? Add before and after pictures portraying what you would like. 
+
+#### The good issue is closed when there i no need for it (except bug reports!)
+<!-- - Close an issue if you do not need it anymore and work haven't begun on it.  -->
+If an issue that you have submitted is no longer necessary and no work has been done on it, go ahead and close it.
+
+This is not because it might not provide value to Kirby. But there is almost certainly more preceeding issues, than for one which there really is no need.
```

Comment:
Two times "not" in this sentence makes it harder to understand.

If I understand it correctly, your point is that while we want to acknowledge that suggestions from contributors are valuable, we need to eliminate "noise" by having issues removed if they are outdated or redundant in some way or another.

I think this comment covers the entire "The good issue is closed when there i no need for it" section.

Could you give it another shot?

Reply:
How's this?
> #### The good issue is closed when there i no need for it (except bug reports!)
> If an issue you have submitted is no longer necessary and no work has been done on it, go ahead and close it.
> 
> While suggestions from contributors always are valueable, we want to eliminate as much "noise" as possible, by removing issues that are outdated or redundant. Otherwise they might take away attention from more important issues.  
> 
> If someone else has a need for your issue again they can request that it is reopened or go ahead and open a new one. No harm done (issues cost nothing to create afterall).                                                                                              
>                                                                                                                                                                                                                                              
>This does not go for bug reports however, as we want to unearth and solve as many bugs as possible.